pkg/volume/host_path/host_path.go

```diff
@@ -132,6 +121,9 @@ func (plugin *hostPathPlugin) NewDeleter(spec *volume.Spec) (volume.Deleter, err
 }
 
 func (plugin *hostPathPlugin) NewProvisioner(options volume.VolumeOptions) (volume.Provisioner, error) {
+	if !plugin.config.ProvisioningEnabled {
```

Comment:
I am highly dubious of the idea of provisioning host paths.  What is the use-case?